monero-harness/src/lib.rs

```diff
@@ -136,11 +136,7 @@ impl<'c> Monero {
         monerod.start_miner(&miner_address).await?;
 
         tracing::info!("Waiting for miner wallet to catch up...");
-        let block_height = monerod.client().get_block_count().await?;
-        miner_wallet
-            .wait_for_wallet_height(block_height)
-            .await
-            .unwrap();
+        miner_wallet.refresh().await?;
```

Comment:
🚧
This is unfortunately not equal and I had problems here: 
* `refresh` asks the monerod for blockheight and refreshes until then
* `wait_for_blockheight` waits until the wallet has seen `block_height`. 

This can be a problem here as monerod might not be finished with initializing the 70 blocks and a request from the wallet to monerod could return anything < 70.

Reply:
> This can be a problem here as monerod might not be finished with initializing the 70 blocks and a request from the wallet to monerod could return anything < 70

But we call this a few lines above:

```rust
let block = monerod.client().generate_blocks(70, &miner_address).await?;
```

Doesn't that mean that we wait for the blocks to be generated before we continue?

Also, all the tests pass on CI, so it would seem like this doesn't cause any problems... If you are very confident that it is necessary we can revert the change. It was just an optimisation anyway.

Reply:
My assumption is it depends on how fast your machine is. Let's leave it as you propose and if it becomes a problem we can revert.